tmpl/package.tmpl

```diff
@@ -107,6 +107,24 @@ func {{.GoName}}({{template "paramsGoDecl" .Parameters}}){{if not .Return.IsVoid
 }
 {{end}}
 
+// Init initializes the OpenGL bindings by loading the function pointers (for
+// each OpenGL function) from the active OpenGL context.
+//
+// It must be called under the presence of an active OpenGL context, e.g. always
+// after calling window.MakeContextCurrent() and always before calling any
+// OpenGL functions exported by this package.
+//
+// On Windows, Init loads pointers that are context-specific (and hence you
+// must re-init if switching between OpenGL contexts, although it may work if
```

Comment:
I don't think so. I think that would make it more confusing. The point is that _it's basically_ OK (but not technically correct as in perfect) to not call `gl.Init` assuming people do not use multiple differing brands of graphics cards (i.e. most users).